eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -132,6 +139,7 @@ data ConwayGovPredFailure era
       (Map.Map (Credential 'ColdCommitteeRole (EraCrypto era)) EpochNo)
   | InvalidPrevGovActionIdsInProposals (Seq.Seq (ProposalProcedure era))
   | VotingOnExpiredGovAction (Map.Map (GovActionId (EraCrypto era)) (Voter (EraCrypto era)))
+  | InvalidPrevGovActionId (ProposalProcedure era)
```

Comment:
You should be replacing the old predicate failure instead. We haven't deployed Conway yet, so we are still free to change predicate failures.
```suggestion
  | InvalidPrevGovActionId (ProposalProcedure era)
  | VotingOnExpiredGovAction (Map.Map (GovActionId (EraCrypto era)) (Voter (EraCrypto era)))
```